.changeset/purple-kiwis-sit.md

```diff
@@ -0,0 +1,7 @@
+---
+'@commercetools-docs/gatsby-theme-docs': major
+'@commercetools-docs/ui-kit': patch
+'@commercetools-website/api-docs-smoke-test': patch
```

Comment:
No releases for smoke test sites please, they are not published to NPM

Reply:
That's ok, changeset includes all packages in the workspace.
Only packages marked as public will be released then.

Reply:
I remember we removed them in the past, but you're right that it does not matter in the end.

Reply:
So I changed now the theme to minor and kept the other two as before 👍